Aptoide/aptoide-client-v8
# [HOTFIX/APP-129] Remove search keywords

**What does this PR do?**

   This PR aims at removing search native ads keywords and adding a new placement ID.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] SearchResultFramgent.java

**How should this be manually tested?**

Search for example Spotify and check if you have native ads. Notice that to test this you will need to use the new ad unit key. If you dont change it, you will be making requests with a adunit as 0.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-129](https://aptoide.atlassian.net/browse/APP-129)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

Yes, we changed the ad unit for search

**Code Review Checklist**
